pkgconfig/CMakeLists.txt

```diff
@@ -62,11 +64,11 @@ set(ENCLAVE_CXXLIBS "${ENCLAVE_CLIBS_1} -loelibcxx ${ENCLAVE_CLIBS_2}")
 
 set(HOST_INCLUDES "-I\${includedir}")
 
-set(HOST_CFLAGS_CLANG "-mllvm -x86-speculative-load-hardening")
+set(HOST_CFLAGS_CLANG "-fstack-protector-strong -mllvm -x86-speculative-load-hardening")
```

Comment:
Sorry for the merge conflict 😢 let me know if you want me to rebase this.

Reply:
Rebased and updated to new style.